test/e2e/info_test.go

```diff
@@ -175,4 +176,27 @@ var _ = Describe("Podman Info", func() {
 		Expect(session).To(Exit(0))
 		Expect(session.OutputToString()).To(Equal(want))
 	})
+
+	It("Podman info: check lock count", func() {
+		// This should not run on architectures and OSes that use the file locks backend.
+		// Which, for now, is Linux + RISCV and FreeBSD, neither of which are in CI - so
+		// no skips.
+		info1 := podmanTest.Podman([]string{"info", "--format", "{{ .Host.FreeLocks }}"})
+		info1.WaitWithDefaultTimeout()
+		Expect(info1).To(Exit(0))
+		free1, err := strconv.Atoi(info1.OutputToString())
+		Expect(err).To(Not(HaveOccurred()))
+
+		ctr := podmanTest.Podman([]string{"create", ALPINE, "top"})
+		ctr.WaitWithDefaultTimeout()
+		Expect(ctr).To(Exit(0))
+
+		info2 := podmanTest.Podman([]string{"info", "--format", "{{ .Host.FreeLocks }}"})
+		info2.WaitWithDefaultTimeout()
+		Expect(info2).To(Exit(0))
+		free2, err := strconv.Atoi(info2.OutputToString())
+		Expect(err).To(Not(HaveOccurred()))
+
+		Expect(free1).To(Equal(free2 + 1))
```

Comment:
This will be hard to debug if the test fails, you should add a description here.
But more importantly this test must be written as system tests or use the `Serial` decorator, due our use of /dev/shm this is not safe in a parallel context.

Reply:
Done, test is now Serial and has an extra comment added.